bundles/WordExportBundle/public/js/startup.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ * @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+pimcore.registerNS("pimcore.bundle.wordexport.startup");
+/**
+ * @private
+ */
+pimcore.bundle.wordexport.startup = Class.create({
+    initialize: function () {
+        document.addEventListener(pimcore.events.preMenuBuild, this.preMenuBuild.bind(this));
+    },
+
+    preMenuBuild: function (e) {
+
+        let menu = e.detail.menu;
+        let that = this;
+        const user = pimcore.globalmanager.get('user');
+        const perspectiveCfg = pimcore.globalmanager.get("perspective");
+
+        if (user.isAllowed("translations") && perspectiveCfg.inToolbar("extras.translations")) {
```

Comment:
Just leaving a reminder, that similarly to XLIFF, it may need to have own separate permissions https://github.com/pimcore/pimcore/pull/13967#discussion_r1064412673